Source/Fuse.Common/Internal/MiniList.uno

```diff
@@ -11,16 +20,26 @@ namespace Fuse.Internal
 	struct MiniList<T> : IList<T> where T : class
 	{
 		object _list;
+		ObjectList<T> AsList { get { return (ObjectList<T>)_list; } }
+		
+		MiniListMode _mode = MiniListMode.Empty;
```

Comment:
Well, this list is supposed to trade performance for size. You undo this here, rendering the semantics pointless. IMO, it would be better to switch away from MiniList<T> and to a normal List<T> in this case.